Tests/PointFreeTests/AccountTests/__Snapshots__/AccountTests/testEpisodeCredits_1Credit_1Chosen.1.Conn.txt

```diff
@@ -1753,6 +1757,11 @@ html {
                       #1: Proof in Functions
                     </a>
                   </li>
+                  <li>
+                    <a href="/episodes/ep1-proof-in-functions" class="pf-link-purple">
+                      #1: Proof in Functions
```

Comment:
huh, this seems to be duplicated now. is this the addition of the paid episode to the mocks? 

maybe we should change more info to distinguish those values.